python/src/iceberg/transforms.py

```diff
@@ -83,10 +83,8 @@ def preserves_order(self) -> bool:
     def satisfies_order_of(self, other) -> bool:
         return self == other
 
-    def to_human_string(self, value) -> str:
-        if value is None:
-            return "null"
-        return str(value)
+    def to_human_string(self, value: Optional[S]) -> str:
+        return str(value) if value is not None else "null"
```

Comment:
Is this change necessary for this PR? We generally try to avoid code churn that isn't making functional changes.